samples/adbserver-sample/build.gradle.kts

```diff
@@ -11,6 +11,6 @@ android {
 dependencies {
     implementation(libs.kotlinStdlib)
     implementation(libs.appcompat)
-    implementation(projects.adbServer.adbserverDevice)
+    implementation("com.kaspersky.android-components:adbserver-device")
```

Comment:
nit: would be cool to use toml

Reply:
toml require to set version, yes in this case we can set any version, but i thing current way is more clear in this situation